pyInterface/module/_config.py

```diff
@@ -171,6 +171,11 @@ def _readBinning(binningString):
 				return []
 	return retval
 
+def _roundSignificantDigits(value, significantDigits):
+	oomValue = int( math.floor(math.log10(abs(value))))
```

Comment:
there is no space to balance the one after `int(`, please write either `int( math.floor(math.log10(abs(value))) )` or `int(math.floor(math.log10(abs(value))))`